package.json

```diff
@@ -4,10 +4,21 @@
   "description": "This is a Workshop about Cypress",
   "main": "index.js",
   "scripts": {
+    "clean:reports": "rm -rf cypress/reports && mkdir cypress/reports",
```

Comment:
you can change this to: `rm -rf cypress/reports/* || true` so in that way the rem -rf will not fail when reports dir does not exist

Reply:
I've updated command and it worked as expected with git bash. Thanks @holgiosalos